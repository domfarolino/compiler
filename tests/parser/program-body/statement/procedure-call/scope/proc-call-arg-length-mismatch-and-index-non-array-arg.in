program dominic is

  integer a;
  procedure fn(integer arg1 in)
  begin
  end procedure;

begin
  fn(a[10], b);
end program.
