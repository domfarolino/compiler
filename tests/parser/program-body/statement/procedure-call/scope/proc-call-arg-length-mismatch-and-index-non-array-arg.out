[1;31mLine 9: Cannot index into non-array name 'a'
Line 9: Error parsing program body
[0m